src/main/java/org/candlepin/subscriptions/subscription/export/SubscriptionDataExporterService.java

```diff
@@ -74,23 +74,22 @@ public class SubscriptionDataExporterService
   static final String PRODUCT_ID = "product_id";
   static final String METRIC_ID = "metric_id";
   static final String CATEGORY = "category";
-  private final Map<String, Function<String, Specification<SubscriptionCapacityView>>>
-      filterFuntions =
-          Map.of(
-              PRODUCT_ID,
-              SubscriptionDataExporterService::handleProductIdFilter,
-              "usage",
-              SubscriptionDataExporterService::handleUsageFilter,
-              CATEGORY,
-              this::handleCategoryFilter,
-              "sla",
-              SubscriptionDataExporterService::handleSlaFilter,
-              METRIC_ID,
-              SubscriptionDataExporterService::handleMetricIdFilter,
-              "billing_provider",
-              SubscriptionDataExporterService::handleBillingProviderFilter,
-              "billing_account_id",
-              SubscriptionDataExporterService::handleBillingAccountIdFilter);
+  private final Map<String, Function<String, Specification<SubscriptionCapacityView>>> filters =
+      Map.of(
+          PRODUCT_ID,
+          SubscriptionDataExporterService::handleProductIdFilter,
+          "usage",
+          SubscriptionDataExporterService::handleUsageFilter,
+          CATEGORY,
+          this::handleCategoryFilter,
+          "sla",
+          SubscriptionDataExporterService::handleSlaFilter,
+          METRIC_ID,
+          SubscriptionDataExporterService::handleMetricIdFilter,
+          "billing_provider",
```

Comment:
Might as well make these two static declarations like the others

Reply:
Spite of using static literals instead of plain strings is a good practice, we're not following this in the project. The other static declarations are there because we're reusing it in other part of the code. 

I personally like the idea of always using static literals, so I think this is a good topic for the retrospective meeting.